templates/SilverStripe/DiscovererSearchUI/Page/Layout/SearchResults.ss

```diff
@@ -3,5 +3,13 @@
         $SearchForm
     </div>
 
+    <% if $QuerySuggestions %>
```

Comment:
if there are no suggestions this is an empty header - maybe it should only display if there are suggestions available?